c2corg_api/views/user.py

```diff
@@ -288,6 +296,146 @@ def post(self):
         return {}
 
 
+class UpdateAccountSchema(colander.MappingSchema):
+    email = colander.SchemaNode(
+            colander.String(),
+            missing=colander.drop,
+            validator=colander.All(
+                colander.Email(),
+                colander.Function(
+                    partial(available_user_attribute, 'email'),
+                    'Already used email'
+                )
+            ))
+    toponame = colander.SchemaNode(
```

Comment:
For all other fields (100+) we use the same name in the database model, in the API and in the UI. Let's please do the same for these two. (And there are also fields with an underscore that apparently haven't been a problem so far with Angular.)